packages/dynamic_forms/lib/src/parser/parser_node.dart

```diff
@@ -90,8 +98,10 @@ abstract class ParserNode {
   static String emptyString() => "";
   static String convertToString(String x) => x;
   static bool convertToBool(String x) => x?.toLowerCase() == "true";
+  static int convertToColor(String x) => int.parse(x);
   static bool defaultFalse() => false;
   static bool defaultTrue() => true;
   static int defaultInt() => 0;
   static Decimal defaultDecimal() => Decimal.fromInt(0);
+  static double defaultDouble() => 2.0;
```

Comment:
> Thanks for the pull request with another great component!
> We actually implemented our own custom slider internally, but it is slightly different because it also has a text input next to the slider but it was too specific to our app so I didn't have time to put it in the component package. But I think it is better to have this slider inside the package first.
> 
> Just one more suggestion: Have you tried to generate the slider using the YAML and the generator? In this case, I think it would save a lot of code.

I didnt try to use YAML, i have to check it :) I have sometimes free time, so I'm adding a manual but I'll look at this suggestion.

Reply:
> the default value for double should probably be 0

Yes i was have  problem with value and minValue so i changed to 2 but then I solved the problem.  This should be 0 ! right.